### PR TITLE
feat: Endpoint to request an invite to an organization's Slack channel by user ID #207

### PR DESCRIPTION
**Pull request by blefebvre**
Ticket #207 

Proposes addition of a new endpoint: `POST /slack/channels/invite-by-user-id`

As defined in the API spec:

image

## Change Management

```yaml
cm-assessment: v1
changeType: standard
impact: unnoticeable
risk: minor
changeApprovedBy: ["Dominique Jäggi"]
rationale: "Auto-added baseline (no PR assessment present at CI time); refine via the cmr skill if this change is higher-impact."
```

### CODE REVIEW

File: src/controllers/slack.js [hunk 1]
```diff
@@ -131,7 +135,73 @@ function SlackController(SlackApp) {
...
+      await elevatedClient.inviteUsersByEmail(orgSlackChannelId, [userProfile]);
+    } catch (error) {
+      log.error(`Error inviting user: ${userProfile.userId} to Slack channel: ${orgSlackChannelId}. Message: ${error.message}`);
+      return internalServerError('Error inviting user to Slack channel.');
+    }
+
+    /*
+    TODO:
+      - Update invitedUserCount in the organization's Slack config
+      - Persist organization back to data store
```
Comment (blefebvre) [inline src/controllers/slack.js:195]: These two items are pending a conversation with our Cloud Manager peers.

File: src/controllers/slack.js [hunk 2]
```diff
@@ -12,8 +12,12 @@
 
 import { Response } from '@adobe/fetch';
 import { cleanupHeaderValue } from '@adobe/helix-shared-utils';
-import { internalServerError } from '@adobe/spacecat-shared-http-utils';
-import { hasText } from '@adobe/spacecat-shared-utils';
+import {
+  badRequest,
+  createResponse,
+  internalServerError, notFound,
```
Comment (solaris007) [inline src/controllers/slack.js:18]: linebreak before _notFound_

**Review by solaris007 — APPROVED** [review]
LGTM

**Pull request merged**